torch/_C/__init__.pyi.in

```diff
@@ -1042,3 +1042,6 @@ def _faulty_agent_init() -> _bool: ...
 def _enable_minidumps(directory: str) -> None: ...
 def _disable_minidumps() -> None: ...
 def _enable_minidumps_on_exceptions() -> None: ...
+
+# Defined in torch/csrc/Runtime.cpp
```

Comment:
Wrong comments?

Reply:
This is not required with new implementation. I will update this.